packages/cosmic-swingset/test/test-make.js

```diff
@@ -49,28 +49,3 @@ test.serial('make and exec', async t => {
   t.log('export');
   t.is(await scenario2.export(), 0, 'export exits successfully');
 });
-
-test.serial('integration test: rosetta CI', async t => {
```

Comment:
If you want to disable the testing of Rosetta, you might consider this one-line change:

```js
// Skipping this test because [...some reason here]
test.skip('integration test: rosettta CI', async t => {
```

Reply:
restoring the file to original